evaluators/text/RougeEvaluator/__init__.py

```diff
@@ -0,0 +1,27 @@
+from typing import Sequence
+
+from rouge import Rouge
+
+from jina.executors.evaluators.text import BaseTextEvaluator
+
+class RougeEvaluator(BaseTextEvaluator):
+    """
+    :class:`RougeEvaluator` Evaluate Rouge score between acutal and ground truth.
+    """
+
+    def __init__(self, metrics: str='rouge-1', stats: str='r', *args, **kwargs):
+        """metrics: can be rouge-1, rouge-2 or rouge-l
+        stats: can be r, p or f
+        """
+        super().__init__(*args, **kwargs)
+        self.metrics = metrics
+        self.stats = stats
+        self.rouge = Rouge(metrics=[metrics], stats=[stats])
```

Comment:
This initialization seems like something we should move to 'post_init', to avoid serializing it